requirements-jax-cuda.txt

```diff
@@ -8,7 +8,7 @@ torch>=2.1.0
 # Jax with cuda support.
 # Keep same version as Keras repo.
 --find-links https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
-jax[cuda12_pip]==0.6.0
+jax[cuda12_pip]==0.6.2
```

Comment:
How is this even working?

https://github.com/jax-ml/jax/blob/008522175ec75f4c2d70f6a6ea56e0d2da9771ed/CHANGELOG.md?plain=1#L69

Reply:
Interesting.  Maybe it's not working?  I didn't actually try this.

Reply:
Yeah, it's never been used I believe, so we wouldn't know if it's broken.